releases/release-1.6/go-signal-reports/2017-03-16.md

```diff
@@ -139,7 +139,7 @@
 - Feature owners submit docs PR which references issue in features repo by **TODAY!**
 - Feature owners prepare release notes by **TODAY!**
 - Address etcd upgrade test failures
-- Begin adding human curated release notes to [the release notes draft](https://github.com/kubernetes/sig-release/blob/release-notes-draft-1.6/release-1.6/release-notes-draft.md) by Friday, 17 March 2017 **please commit directly to the `release-notes-draft-1.6` branch so we can comment on the [associated PR](https://github.com/kubernetes/features/pull/203) collectively** 
+- Begin adding human curated release notes to [the release notes draft](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.6/release-notes-draft.md) by Friday, 17 March 2017 **please commit directly to the `release-notes-draft-1.6` branch so we can comment on the [associated PR](https://github.com/kubernetes/features/pull/203) collectively** 
```

Comment:
Might be a good idea to link to a dedicated commit to let the link never fail in the future.